org.opent2t.onboarding.hue/js/common.js

```diff
@@ -0,0 +1,22 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+
+class accessTokenInfo {
+    constructor(
+        accessToken,
+        refreshToken,
+        tokenType, // ex: 'bearer'
```

Comment:
I've seen "e.g." and "ex." being used, though "ex." seems to be less formal. Other files uses ex. as well, should we change all to e.g. ?